### PR TITLE
feat: Generated the Webex provider

### PR DESCRIPTION
**Pull request by TheOjasSingh**
### What change does this PR introduce?

Created the Webex Chat provider

### Why was this change needed?

The Webex chat provider is a versatile and useful tool for individuals and teams in various settings. It is a secure and reliable way to communicate, collaborate, and share information.

### CODE REVIEW

**Comment by Cliftonz** [comment]
#4379

File: providers/webex-chat/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Novu WebexChat Provider
+
+A WebexChat chat provider library for [@novu/node](https://github.com/novuhq/novu)
+
+## Usage
+
+```javascript
+    FILL IN THE INITIALIZATION USAGE
```
Comment (Cliftonz) [inline providers/webex-chat/README.md:8]: Please add the example usage here.

File: providers/webex-chat/package.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@novu/webex-chat",
+  "version": "0.16.3",
```
Comment (Cliftonz) [inline providers/webex-chat/package.json:3]: This should be the latest version 0.20 something

**Review by Cliftonz — CHANGES_REQUESTED** [review]
Please make sure to include tests in this PR too, Thank you for your contribution!

File: providers/webex-chat/package.json (hunk 2 above)
Comment (TheOjasSingh) [reply providers/webex-chat/package.json:3]: This is the most recent version.

File: providers/webex-chat/README.md (hunk 1 above)
Comment (TheOjasSingh) [reply providers/webex-chat/README.md]: added in the recent commit

**Review by TheOjasSingh — COMMENTED** [review]
Added the changes in the recent commit

**Pull request closed without merging**